### PR TITLE
#11249: custom selected class

### PR DESCRIPTION
**Pull request by Erwane**
allow custom selected class for radio and checkbox label.

```php
$this->loadHelper('Form', [
    'templates' => [
        'selectedClass' => 'active',
    ],
]);
```

related to #11249

### CODE REVIEW

**Comment by Erwane** [comment]
missing templates in tests. I will fix that.

File: src/View/Widget/MultiCheckboxWidget.php [hunk 1]
```diff
@@ -210,7 +210,8 @@ protected function _renderInput($checkbox, $context)
             ];
 
             if ($checkbox['checked']) {
-                $labelAttrs = $this->_templates->addClass($labelAttrs, 'selected');
+                $selectedClass = $this->_templates->format('selectedClass', []);
```
Comment (markstory) [inline src/View/Widget/MultiCheckboxWidget.php]: What happens when a user is using a custom set of templates that doesn't define `selectedClass`?

Comment (Erwane) [reply src/View/Widget/MultiCheckboxWidget.php]: I'm not sure it is relevant.
If the selectedClass is not included in custom set of templates, the stringTemplate will thrown an exception, like all missing required templates.

The selectedClass is set to `selected` by default in the `FormHelper`

Comment (markstory) [reply src/View/Widget/MultiCheckboxWidget.php]: Wouldn't that template missing error cause applications to break when they update?

Comment (ADmad) [reply src/View/Widget/MultiCheckboxWidget.php]: @markstory He has added the new template in FormHelper, that's where all templates used by widgets are set.

**Pull request merged**